.github/workflows/docker-build-push.yaml

```
@@ -0,0 +1,35 @@
+name: docker-build-push
+
+on:
+  push
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
```

we're locking  to specific commits in other repos, please do that as well here:

```suggestion
        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```